src/pages/Dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,38 @@
+export const DashboardHeader = () => (
+  <div className='flex flex-col p-8 lg:p-10 justify-center items-center gap-6 self-stretch'>
+    <div className='text-primary transition-all duration-300 text-center text-3xl xxs:text-5xl lg:text-6xl font-medium'>
+      Welcome to dApp Template
+    </div>
+
+    <div className='text-secondary transition-all duration-300 text-center text-base xxs:text-lg lg:text-xl font-medium'>
+      The MultiversX dApp Template, built using{' '}
+      <a
```

Comment:
This component is repeated here thrice, let's have it as a component of `DashboardHeader` and it should receive `children` and `href` as properties.